cloud/pkg/cloudhub/servers/httpserver/signcerts.go

```diff
@@ -39,7 +39,8 @@ func SignCerts() ([]byte, []byte, error) {
 		Organization: []string{"KubeEdge"},
 		Usages:       []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		AltNames: certutil.AltNames{
-			IPs: getIps(hubconfig.Config.AdvertiseAddress),
+			DNSNames: hubconfig.Config.DNSNames,
```

Comment:
nit: DNSNames => DomainNames ?

Reply:
> nit: DNSNames => DomainNames ?

@kevin-wangzefeng Thank for your suggestion. DNSNames is a bit weird, but in k8s, it is named like this, so we'd better keep using DNSNames.